DigitalLearningSolutions.Data/Models/CourseSettings.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace DigitalLearningSolutions.Data.Models
+{
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+
+    public class CourseSettings
+    {
+        public readonly bool ShowPercentage = true;

```

Comment:
Minor but using a property with getter only is more consistent with other classes I think 🙂

Reply:
Yep, sorry! Fixed in ea68e57 🙂